CMakeLists.txt

```diff
@@ -91,6 +95,25 @@ list(APPEND uv_cflags ${lint-no-hides-global-msvc})
 list(APPEND uv_cflags ${lint-no-conditional-assignment-msvc})
 list(APPEND uv_cflags ${lint-no-unsafe-msvc})
 
+check_include_file(sys/eventfd.h UV_HAVE_EVENTFD_H)
+check_include_file(sys/statvfs.h UV_HAVE_STATVFS_H)
+check_include_file(sys/statfs.h UV_HAVE_STATFS_H)
+
+#check_function_exists(eventfd UV_HAVE_EVENTFD)
```

Comment:
Why this is commented out?

Reply:
sry, forgot remove this after test new check